### PR TITLE
Bug fixes in all pages

### PR DESCRIPTION
**Pull request by Pallavi-PH**
This PR will:

1. Alignment issues in the `Community page` footer
Screenshot 2021-06-15 at 7 33 42 PM

2. Places button at a fixed place in each cards on `Commercial support` page
Screenshot 2021-06-15 at 7 35 15 PM

3. Fixes tab alignment and styling on the Blog page
Screenshot 2021-06-15 at 7 36 21 PM

4. Add missing footer background image in Blog page
Screenshot 2021-06-15 at 7 38 02 PM

5. Provide links to Blog > Author page breadcrumbs
6. Fix header styling in Blog > Author page (spacing and background image, author image size)
Screenshot 2021-06-15 at 7 39 58 PM 
7. Fix pagination styling on Blog page
Screenshot 2021-06-15 at 7 41 39 PM

8. Fix tab styling in FAQ page
Screenshot 2021-06-15 at 7 43 17 PM

9. Adjust spacing around the question on FAQ page according to design
10. Add sponsorers section on FAQ page
11. Add missing footer background image on FAQ page
12. Assign correct font colors in inner pages
13. Fix alt texts in community page
14. Maintain same spacing around content across inner pages

### CODE REVIEW

File: website/src/pages/PrivacyPolicy/style.ts [hunk 1]
```diff
@@ -8,7 +8,7 @@ const useStyles = makeStyles((theme) => ({
         },
     },
     pageSpacing: {
```
Comment (rakeshPRaghu) [inline website/src/pages/PrivacyPolicy/style.ts:10]: is `pageSpacing` used as a container? if thats the case i think its better to use max-width

File: website/src/pages/Faq/styles.ts [hunk 2]
```diff
@@ -24,30 +24,43 @@ const useStyles = makeStyles((theme: Theme) => ({
...
   headingSelected: {
-    fontSize: "pxToRem(15)",
+    fontSize: "16px",
     fontWeight: 700,
   },
   accordionRoot: {
     width: "100%",
     [theme.breakpoints.down("xl")]: {
-      padding: theme.spacing(5, 32, 0),
+      padding: theme.spacing(5, 39, 0),
```
Comment (rakeshPRaghu) [inline website/src/pages/Faq/styles.ts:62]: same here

File: website/public/getPosts.js [hunk 3]
```diff
@@ -9,6 +9,7 @@ const getPosts = () => {
         if (err) {
             console.error('Failed to load files from the directory' + err);
         }
+
```
Comment (satyapriyamishra222) [inline website/public/getPosts.js:12]: Why this file is dirty ?

Comment (Pallavi-PH) [reply website/public/getPosts.js]: Removed additional empty line

File: website/src/pages/PrivacyPolicy/style.ts (hunk 1 above)
Comment (Pallavi-PH) [reply website/src/pages/PrivacyPolicy/style.ts:10]: Good one! Replaced padding with width.

File: website/src/pages/Faq/styles.ts (hunk 2 above)
Comment (Pallavi-PH) [reply website/src/pages/Faq/styles.ts]: Replaced padding with width.

**Review by satyapriyamishra222 — APPROVED** [review]

File: website/src/pages/Blog/styles.ts [hunk 4]
```diff
@@ -8,58 +8,75 @@ const useStyles = makeStyles((theme: Theme) => ({
...
   },
   authorText: {
     display: "flex",
     flexWrap: "wrap",
     textAlign: "center",
-    margin: theme.spacing(3, 0),
+    margin: theme.spacing(0),
     justifyContent: "center",
     alignItems: "center",
+    fontSize: '42px'
```
Comment (rakeshPRaghu) [inline website/src/pages/Blog/styles.ts:74]: shouldn't we add a responsive fontsize for this one

File: website/src/pages/Community/index.tsx [hunk 5]
```diff
@@ -130,45 +135,54 @@ const Community: React.FC = () => {
...
+                <img
+                  src="../Images/logos/gostor.svg"
+                  alt={t("generic.gostor")}
+                  className={classes.sponsorCompany}
+                />
+                <img
+                  src="../Images/logos/openzfs.svg"
+                  alt={t("generic.openzfs")}
+                  className={classes.sponsorCompany}
+                />
```
Comment (rakeshPRaghu) [inline website/src/pages/Community/index.tsx:182]: this can taken up as a refactor, get the sponsor list from a json

Comment (Pallavi-PH) [reply website/src/pages/Community/index.tsx]: Done

File: website/src/pages/Blog/styles.ts (hunk 4 above)
Comment (Pallavi-PH) [reply website/src/pages/Blog/styles.ts]: We had added separate font size for mobile devices as well. Now converted both font-size units in `rem`

**Review by rakeshPRaghu — APPROVED** [review]
LGTM

**Review by AVRahul — APPROVED** [review]

**Pull request merged**